packer/scripts/firstuser.sh

```diff
@@ -28,7 +28,8 @@ startServices() {
     pids=`pidof node`
     if [ `expr length "$pids"` -eq "0" ]; then
        echo "starting rackhd ./src services..."
-       sudo nf start > /tmp/rackhd.log &
+       sudo pm2 start rackhd.yml 
+       sudo pm2 logs > /tmp/rackhd.log &  
```

Comment:
@yyscamper Thanks.

Run ```sudo pm2 logs ``` before ```pm2 start``` can solve this plobrem.

But  I'll delete this logs operation, it's useless here.